### PR TITLE
Changes the Eddiehub's discord name and link

### LINKED ISSUE #46: fix: README.md mentions our Discord server's previous name.
this

### LINKED ISSUE #48: Current EddieHub's Discord link is invalid
**Describe the bug**
In the Readme.md, when we click on link to join EddiHub on Discord then it shows the invalid invite link.

**To Reproduce**
1. Go to 'README.md' file
2. Scroll down to 'About EddieBotUI'
3. Click on 'click to join'

**Expected behavior**
It should direct everyone to the Discord's website where we should see invite link of EddieHub's Discord server

**Screenshots**
eddie discord

**Desktop (please complete the following information):**
 - OS: All OS
 - Browser : All Browsers
 - Version : All Versions

**Smartphone (please complete the following information):**
 - Device: All Devices
 - OS: All OS
 - Browser: All Browsers
 - Version: All Versions

**Additional context**

### PR DESCRIPTION
**Pull request by ShaileshKumar007**
This PR fixes issues #46  and #48  by changing the previous name to current name and invalid invite link to valid invite link.
closes #46 and closes #48

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
 ## About EddieBotUI
 
-EddieBotUI is a web application built to display all kinds of information about the **Eddie Jaoude Discord community** - [click to join](https://discord.gg/fPss8W9) - like users biography, usage stats, badges and more.
+EddieBotUI is a web application built to display all kinds of information about the **EddieHub Discord community** - [click to join](https://discord.com/invite/jZQs6Wu) - like users biography, usage stats, badges and more.
```
Comment (BOLT04) [inline README.md:33]: How about we use the same invite link as the other repos (this is related to [this issue](https://github.com/EddieHubCommunity/EddieHubCommunity.github.io/issues/163#issuecomment-826338069))? This way we don't need to change when the link is invalid 🙂 
```suggestion
EddieBotUI is a web application built to display all kinds of information about the **EddieHub Discord community** - [click to join](http://discord.eddiehub.org/) - like users biography, usage stats, badges and more.
```

**Review by BOLT04 — COMMENTED** [review]
Thank you for contributing @ShaileshKumar007 this looks great 👍, just left a comment so let me know what you think. 
Also, you can add `closes #46 ` and `closes #48` to link this PR to those issues. Then when it's merged they will close automatically.

**Review by KukretiShubham — APPROVED** [review]
after suggestions by @BOLT04 LGTM 🚀

File: README.md (hunk 1 above)
Comment (ShaileshKumar007) [reply README.md:33]: That's awesome. With this, we don't have to change the link again and again.
Thanks for helping me out  :-)

Comment (suitupstan) [reply README.md:33]: @ShaileshKumar007 please merge this change? 🤔

**Review by suitupstan — CHANGES_REQUESTED** [review]

**Comment by eddiejaoude** [comment]
Thank you for your contribution! This PR has now been superseded, so I will close

**Pull request closed without merging**